### PR TITLE
make it possible to set `noindex, nofollow` for blog entries

### PR DESCRIPTION
**Pull request by ndzoesch**

### CODE REVIEW

File: source/_layouts/default.twig.html [hunk 1]
```diff
@@ -12,7 +12,9 @@
     {% endblock %}
 
     {% if page.menu_chapter %}
```
Comment (janbuecker) [inline source/_layouts/default.twig.html]: Hmm.. I think that this IF is quite confusing as you don't check on `robots.hide`, only in the else block.

Comment (ndzoesch) [reply source/_layouts/default.twig.html]: I know what you mean. The if was there before, and I did not want to risk having two meta tags with the same content. Any idea how to solve this more elegant? @janbuecker

Comment (janbuecker) [reply source/_layouts/default.twig.html]: What about this?

```
{% if page.menu_chapter %}
    {% if page.robots.hide == true %}
        <meta name="robots" content="noindex, nofollow" />
    {% else %}
        <meta name="robots" content="noindex" />
    {% endif %}
{% endif %}
```

Comment (ndzoesch) [reply source/_layouts/default.twig.html]: Won't work, since blog post do not have the page.menu_chapter property. 😐

**Pull request merged**